superset/assets/visualizations/deckgl/layers/arc.jsx

```diff
@@ -1,16 +1,26 @@
 import { ArcLayer } from 'deck.gl';
 
-export default function arcLayer(formData, payload) {
+import * as common from './common';
+import sandboxedEval from '../../../javascripts/modules/sandbox';
+
+export default function arcLayer(formData, payload, slice) {
   const fd = formData;
   const fc = fd.color_picker;
-  const data = payload.data.arcs.map(d => ({
+  let data = payload.data.arcs.map(d => ({
     ...d,
     color: [fc.r, fc.g, fc.b, 255 * fc.a],
   }));
 
+  if (fd.js_datapoint_mutator) {
+    // Applying user defined data mutator if defined
+    const jsFnMutator = sandboxedEval(fd.js_datapoint_mutator);
+    data = data.map(jsFnMutator);
```

Comment:
With my recent PR we moved to `js_data_mutator` instead of `js_datapoint_mutator` and the call goes `data = jsFnMutator(data);`